libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Actions/Microsoft.BeginDialog.schema

```diff
@@ -1,30 +1,33 @@
 {
     "$schema": "https://raw.githubusercontent.com/microsoft/botbuilder-dotnet/4.Future/schemas/component.schema",
     "$role": "unionType(Microsoft.IDialog)",
-    "title": "Begin Dialog",
-    "description": "Action which begins another dialog (and when that dialog is done, it will return the caller).",
+    "title": "Call a dialog",
```

Comment:
>"Call a dialog [](start = 13, length = 14)

are we really going to deviate from BeginDialog, are we going to rename OnBeginDialog?

Reply:
Yes. Begin Dialog is super confusing - it can be read as a noun and an action. 
Besides - beginning a dialog may easily be interpreted as beginning new, from scratch, while  calling a dialog much better represents the modular way of building conversations we're trying to instill (vs wide flat all inclusive dialogs)